ref/supported-codecs.md

```diff
@@ -2,10 +2,12 @@ Anbox Cloud combines both software and hardware video encoding in order to utili
 
 Not all codecs are supported by one or more of the supported GPU models, neither are they performing suitably for low latency. Hence, the list of supported video codecs is limited.
 
-Currently, the following video codecs are supported:
+The supported video codecs are:
 
  * H.264 - The use of H.264 requires a license from the [MPEG LA](https://www.mpegla.com/). Ensure you have the rights to stream H.264 encoded video content to your users.
  * AV1 - Depending on the GPU model used in the deployment, AV1 hardware encoding is the default preference over H.264 on supported GPUs, such as NVIDIA Ada Lovelace Architecture-based GPUs like L4. Otherwise, it will fallback to AV1 software encoding.
  * VP8
 
+Anbox Cloud currently supports the Opus audio codec.
```

Comment:
I am not quite sure I want an explicitly supported list for audio codec. Since we're using the builtin audio encoder/decoder from webrtc stack. So strictly speaking, [the builtin audio codecs](https://cs.android.com/android/platform/superproject/main/+/main:external/webrtc/api/audio_codecs/builtin_audio_encoder_factory.cc;l=58;drc=c2a43533197f3f73aab66af0fcdc8bbb1b60747e;bpv=1;bpt=1) are:
-   Opus
-   G722
-   iLBC
-   G711

And the opus codec is preferred if the codec is supported on both ends

Reply:
@adglkh Based on what we discussed in SU and what you investigated further, are you fine with pushing this change as is?

Reply:
@keirthana yes, I'm fine with this.